### ISSUE TITLE
Is the Ghosttree on the forest world beatable?

### ISSUE THREAD

**Issue by mikeweilgart** [SuperTux/supertux#1116]
Using version 0.5.1 (that's what's in the apt sources by default on Ubuntu 18.04).

My son, who is almost six years old, absolutely LOVES Supertux.  So first of all I just want to thank all the developers and the designers for making this game so much fun and such a great and playable game.  (I know this isn't "the place" for that, but seriously, he LOVES it and I thought you'd enjoy hearing that.)

He has played and beat the entire Forest World (except a couple optional levels like Countercurrent that he found too hard), and finally reached the boss on the top of the castle in the level "The Silent Walls."

I couldn't figure out how to beat the ghosttree, so I checked around online and found https://github.com/SuperTux/wiki/blob/master/Forest-Badguys.md#boss-ghost-tree, which says that the ghost tree is not added yet (but it has been in the version I have, apparently) and also says as part of the design ideas:

> You destroy the tree by throwing something in a different color at him (lamps or arcorns?)

But, as I understand from various other places I read about Supertux development plans, the Forest World was removed from story mode in the first place because it's considered unstable (?) and not ready for "prime time."  (Incidentally, it has been completely stable for all of my son's playing all the way through it, and he enjoys it even more than the main story mode, by a long shot.)

I tried throwing different colored lanterns at the ghost tree and it didn't seem to do anything.  Also, in contradiction to the design description, I couldn't tell what color he is supposed to be glowing currently at all.

So, bottom line:

1. Is the ghost tree on the last level of the Forest World actually beatable in Supertux 0.5.1?  (If so, how?)
2. If not, is it beatable in the current development version of the app or any version later than 0.5.1?
3. What are the future plans for Forest World and its inclusion into the main story mode?  (Or where can I go for up to date plans about this?)

(I've also suggested to my son that maybe he will learn to program someday and then modify the game to make the ghost tree work the way it's supposed to.  ;)  He understands about what open source means already.)

**Comment by weluvgoatz** [comment]
I see your son and you are very enthusiastic about SuperTux. That's great! :)
1. The Ghost Tree _is_ beatable, you must collect a wisp in a lantern, and when the ghost tree is sucking in the wisps of a SEPARATE color, you throw the lantern into its mouth.  If this doesn't work, it may be a glitch.
2. 0.6 has been released and contains the forest in the story mode. However, it is not complete and the story mode now stops right before you reach the ghost forest's landscape. Soon, the forest might be finished and may include the ghost tree.

And the SuperTux teams seems to always be looking for more developers ;)

**Comment by mikeweilgart** [comment]
I was missing the aspect that the lantern needs to be thrown in his *mouth,* not simply thrown *at him* like we were trying.  It is indeed beatable; no glitch!  Thanks from my son and me.  :)  Closing issue.
